conda/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set data= load_setup_py_data() %}
+package:
+  name: "ts-atbuilding-vents"
+  version: {{ data.get('version') }}
+
+source:
+  path: ../
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install --no-deps --ignore-installed .
+
+test:
+  requires:
+    - ts-conda-build =0.4
+    - ts-tcpip
+    - ts-utils
+    - ts-xml
+    - smbus3
+    - pymodbus
```

Comment:
Please, pin the version of `pymodbus` at the very least provide a major version pin, but this library has been shown to break from minor releases so you might want to pin major.minor.